flaml/automl.py

```diff
@@ -2376,7 +2376,7 @@ def custom_metric(
         def cv_score_agg_func(val_loss_folds, log_metrics_folds):
             return metric_to_minimize, metrics_to_log
         ```
-                “val_loss_folds” - list of floats, the loss scores of each fold; “log_metrics_folds” - list of dicts/floats, the metrics of each fold to log.
+            “val_loss_folds” - list of floats, the loss scores of each fold; “log_metrics_folds” - list of dicts/floats, the metrics of each fold to log.
```

Comment:
Not sure about this change. Are you able to compile the doc now?

Reply:
![image](https://user-images.githubusercontent.com/5702457/195948774-4c9cfc6e-72dc-486f-9e2a-f639f36903ec.png)

Reply:
Please find the compiled website above.

Reply:
"the following signature" -> "the following input arguments"?
"This function should" should not be in the same line as "log_metrics_folds".